requirements.txt

```diff
@@ -20,3 +20,5 @@ jmespath==0.10.0
 textract
 html5lib==1.1
 click==8.0.2
+pyarrow==8.0.0
+prefect==1.2.2
```

Comment:
Not sure why this wasn't included before. 🤷

Reply:
It looks like originally these were installed during the VM setup, https://github.com/covid-projections/can-scrapers/blob/main/services/prefect/setup_gcp_instance.sh#L57 but I think it definitely makes more sense to have them here in the requirements